lib/bundler/cli/add.rb

```diff
@@ -2,23 +2,38 @@
 
 module Bundler
   class CLI::Add
-    def initialize(options, gem_name)
-      @gem_name = gem_name
+    def initialize(options, gems)
+      @gems = gems
       @options = options
       @options[:group] = @options[:group].split(",").map(&:strip) if !@options[:group].nil? && !@options[:group].empty?
     end
 
     def run
+      # raise error when no gems are specified
+      raise InvalidOption, "Please specify gems to add." if @gems.empty?
+
+      # show warning when options are used with mutiple gems
+      # for time being they will be applied to the last gem
+      Bundler.ui.warn "All the options will be applied to all gems`."
+
       version = @options[:version].nil? ? nil : @options[:version].split(",").map(&:strip)
 
       unless version.nil?
         version.each do |v|
           raise InvalidOption, "Invalid gem requirement pattern '#{v}'" unless Gem::Requirement::PATTERN =~ v.to_s
         end
       end
-      dependency = Bundler::Dependency.new(@gem_name, version, @options)
 
-      Injector.inject([dependency], :conservative_versioning => @options[:version].nil?) # Perform conservative versioning only when version is not specified
+      # to store dependencies formed from gem names
+      dependencies = []
+
+      # create a dependency from gem name and
+      # push to dependencies
+      @gems.each {|g| dependencies << Bundler::Dependency.new(g, version, @options) }
```

Comment:
Might be nicer as
```ruby
dependencies = @gems.each_with_object([]) {|g,acc| acc << Bundler::Dependency.new(g, version, @options) }
```

The comment seems a but redundant as the line isn't super complex as well.
Same with the `inject the dependencies` comment since the line is actually `Inject.inject(dependencies...)`

Reply:
Sure. Will make the changes.